server/views/pages/activities/activitiesMigration.njk

```diff
@@ -180,8 +181,3 @@
   </div>
 
 {% endblock %}
-
-{% block bodyEnd %}
-  {{ super() }}
-  <script src="/assets/filter-toggle-button.js"></script>
```

Comment:
was intending on not removing this javascript, but couldn't get it to work so ended up rewriting